src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -223,6 +230,7 @@ public static PackageMetadataConfig createFromString(String yamlContents) {
         .packageName(buildMapWithDefault((Map<String, String>) configMap.get("package_name")))
         .shortName((String) configMap.get("short_name"))
         .packageType(PackageType.of((String) configMap.get("package_type")))
+        .generationLayer(GenerationLayer.of((String) configMap.get("generation_layer")))
```

Comment:
I think this could make sense, so long as we don't want to generate grpc stubs for core/common proto packages, but I think that we do want to do this so having them be independent settings makes sense. If we don't want to publish the stubs, then I think we can combine them.